dateparser/custom_lang_detect/lang_detect.py

```diff
@@ -0,0 +1,17 @@
+import langdetect
+from dateparser.conf import apply_settings
+
+langdetect.DetectorFactory.seed = 0
+
+
+@apply_settings
+def detect_languages(text, settings=None):
+    language_codes = []
+    try:
+        parser_data = langdetect.detect_langs(text)
+        for langauge_candidate in parser_data:
+            if langauge_candidate.prob > settings.LANGUAGE_DETECTION_CONFIDENCE_THRESHOLD:
+                language_codes.append(langauge_candidate.lang)   
+    except langdetect.lang_detect_exception.LangDetectException:
+        print("langdetect parsing error")
```

Comment:
We have two options here:
* Re-raise the exception with another type of exception (maybe we can create a new exception)
* Ignore / log the exception.

In any case, printing shouldn't be the final result.